pom.xml

```
@@ -95,6 +95,12 @@
             <version>${assertj.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>5.9.2</version>
```

Вот где ошибка, и почему не работает Method Source
```suggestion
            <version>${junit.version}</version>
```

---

У тебя там несовместимые версии